src/components/Modal/node/SendMessageModal.tsx

```diff
@@ -147,14 +159,29 @@ export const SendMessageModal = (props: SendMessageModalProps) => {
 
   const handleCloseModal = () => {
     set_sendMode('directMessage');
-    set_numberOfHops('');
+    set_numberOfHops(0);
     set_message('');
     set_receiver(props.peerId ? props.peerId : '');
     set_path('');
     set_openModal(false);
     set_status('');
   };
 
+  const handlePathKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === ' ' || e.key === 'Spacebar') {
+      e.preventDefault(); // Prevent space key from inserting a space
+      const input = e.target as HTMLInputElement;
+      // These are always a number... never null
+      const start = input.selectionStart!;
+      const end = input.selectionEnd!;
+      set_path((prevPath) => prevPath.substring(0, start) + '\n' + prevPath.substring(end));
+      // Set a timeout to ensure that the focus and blur events have a chance to complete
+      setTimeout(() => {
+        input.setSelectionRange(start + 1, start + 1);
+      }, 0);
+    }
```

Comment:
```suggestion
    else if (e.key === ',' ) {
      e.preventDefault(); // Prevent space key from inserting a space
      const input = e.target as HTMLInputElement;
      // These are always a number... never null
      const start = input.selectionStart!;
      const end = input.selectionEnd!;
      set_path((prevPath) => prevPath.substring(0, start) + ',\n' + prevPath.substring(end));
      // Set a timeout to ensure that the focus and blur events have a chance to complete
      setTimeout(() => {
        input.setSelectionRange(start + 1, start + 1);
      }, 0);
    }
```

Reply:
also add new line for `,`
This should work, please test

Reply:
Selection range had to be adjusted, but good for everything else! 👍🏻